### DISCUSSION TITLE
🥳 Check Out Our New Logo Design! 🎊🎉

### DISCUSSION THREAD

**Discussion by daley-mottley** [dmotts/yt-to-cloud-uploader#30] [Announcements]
In case you are not aware or have not seen, I would like to announce that we have a new logo!🥳

![Logo-1](https://github.com/user-attachments/assets/dae2c690-18c7-4943-b7db-ba3e68818b51)

Thanks to @shawaiz-khan! 
We appreciate your hard work! 💪😁

**Comment by shawaiz-khan** [comment]
Thank You

**Comment by HomerusJa** [comment]
Looks great! @shawaiz-khan , are you a UI/UX guy? And just wondering: how did you design this and which tools did you use?

**Reply by shawaiz-khan** [reply]
> Looks great! @shawaiz-khan , are you a UI/UX guy? And just wondering: how did you design this and which tools did you use?

Yeah I'm a UI UX person and a frontend web developer! I've used Adobe illustrator for this

**Reply by HomerusJa** [reply]
@shawaiz-khan If you are interested, join #31 as we are discussing the different options to build a frontend there!

**Reply by shawaiz-khan** [reply]
Sure
